docs/index.md

```diff
@@ -16,68 +14,10 @@ Or a video workflow from left to right:
 
 This check uses the Icinga Business Process module to quickly show the overall path is OK, even if one of the paths is having problems.
 
-
 ## Getting Started
 
 Once Meerkat is installed and configured, you simply browse it, make a new dashboard, upload a background, and start dragging and dropping elements over the background. You then give the 'view' link to users to put on TVs on the wall, and you have a ready made NOC.
 
-
-### Deployment with Docker
-Deploying from the docker hub image is the easiest way to get started.
-
-Copy `config/meerkat.toml.example` to `config/meerkat.toml`, edit `config/meerkat.toml` using your favourite editor, then invoke your favourite composer:
-```
-docker-compose up
-```
-or
-```
-docker stack deploy -c docker-compose.yml meerkat
-```
-
-The default `docker-compose.yml` uses the directories `config/` `dashboards/` and `dashboards/` for data persistance.  Feel free to edit docker-compose.yml as you see fit though, of course.
-
-We have also included a Dockerfile in case you want to build your own image.
-
-### Build without Docker
```

Comment:
@ollytom Did you mean to remove this bit?

Reply:
Yes - this is already in the README (for now!)